Abonament.java

```diff
@@ -74,14 +74,16 @@ public static void ab(String Abo, Abonament abonament) {
     //metoda care ajuta utilizatorul sa aleaga abonamentul
     public static void alegeAb() {
         System.out.println(color.GREEN + "\nList of subscriptions:" + color.RESET);
-        System.out.println("\nStart 100" +
-        "\nMax 140" +
-        "\nMax 175" +
-        "\nMax 200" +
-        "\nMax 290" +
-        "\nChoose a subscription: ");
+        String text = """
+                \nStart 100
+                Max 140
+                Max 175
+                Max 200
+                Max 290
+                Choose a subscription: """;
+        System.out.println(text);
         abY = Main.input.nextLine();
-        if (!Objects.equals(abY, "Start 100") && !Objects.equals(abY, "Max 140") && !Objects.equals(abY, "Max 175") && !Objects.equals(abY, "Max 200") && !Objects.equals(abY, "Max 290")){
+        if (!Objects.equals(abY, "Start 100") && !Objects.equals(abY, "Max 140") && !Objects.equals(abY, "Max 175") && !Objects.equals(abY, "Max 200") && !Objects.equals(abY, "Max 290")) {
             System.out.println(Main.color.RED + "\nSuch subscription does not exist!" + Main.color.RESET);
```

Comment:
aici ar fi mai comod daca in loc de sa faci check de multe ori conditiile, sa creezi o lista cu max 100 max 140 etc, si sa faci doar un check daca valoare lui abY se contine in aceasta lista